MAINTAINERS.md

```diff
@@ -1,24 +1,24 @@
 # Project Maintainer
 
 ## How to become a maintainer
-The following are the criteria to get considered becoming a maintainer:
+The following are the criteria to be considered becoming a maintainer:
```

Comment:
I would rephrase it to:
`To be considered a maintainer, the following criteria apply:`

wdyt?